cmd/convert.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kong/deck/convert"
+	"github.com/spf13/cobra"
+)
+
+var (
+	convertCmdSourceFormat      string
+	convertCmdDestinationFormat string
+	convertCmdInputFile         string
+	convertCmdOutputFile        string
+)
+
+// convertCmd represents the convert command
+var convertCmd = &cobra.Command{
+	Use:   "convert",
+	Short: "Convert files in one format to another format",
+	Long: `Convert command converts files representing configuration in one format
+to another compatible format.
+and usage of using your command. For example: a configuration for 'kong-gateway'
```

Comment:
Was something lost here? It looks like there was originally some more sentence before the "and usage..." here.

Reply:
> and usage of using your command

This was a boilerplate that was left. Updated.